pkg/cluster/compute/affinityrule/affinityrule.go

```diff
@@ -97,3 +121,13 @@ func GetAffinityRule(whost *host.WmiHost, affinityRuleName string) (caffinityRul
 	caffinityRule = &AffinityRule{wmiafRule}
 	return
 }
+
+func isSoftAntiAffinitySupported(affinityRule *AffinityRule) bool {
```

Comment:
This is not the right Pattern. Follow other methods in this repo

func (af *AffinityRule) IsSoftAntiAffinitySupported() (issupported bool, err error) {
}

Reply:
Refactored the logic, can you please review again ?